ynr/apps/uk_results/management/commands/uk_results_create_file.py

```diff
@@ -88,7 +88,7 @@ def handle(self, **options):
                 row["person_id"] = membership.person.pk
                 row["person_name"] = membership.person.name
                 row["ballots_cast"] = membership.result.num_ballots
-                row["is_winner"] = membership.result.is_winner
+                row["elected"] = bool(membership.elected)
```

Comment:
I don't think we want to call `bool` here. Consider the values of `elected`:

`False`: Results are in and this person was not elected
`True`: Results are in and this person was elected
`None`: Results are not in

By casting `None` to `False` it might look like we have results for this membership when we don't.

Reply:
Good spot! I've updated this commit to remove the use of bool